### PR TITLE
KEYCLOAK-18913 Update messages_cs.properties

### PR DESCRIPTION
**Pull request by McLaynV**
Czech translations for base login theme

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 1]
```diff
@@ -12,6 +13,11 @@ doDecline=Zamítnout
 doForgotPassword=Zapomenuté heslo?
 doClickHere=Klikněte zde
 doImpersonate=Zosobnit
+doTryAgain=Zkusti znovu
```
Comment (dklika) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:16]:
```suggestion
doTryAgain=Zkusit znovu
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 2]
```diff
@@ -40,6 +46,7 @@ codeErrorTitle=Kód chyby\: {0}
 termsTitle=Smluvní podmínky
 termsTitleHtml=Smluvní podmínky
 termsText=<p>Smluvní podmínky, které se mají definovat</p>
```
Comment (dklika) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:48]:
```suggestion
termsText=<p>Smluvní podmínky k odsouhlasení</p>
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 3]
```diff
@@ -40,6 +46,7 @@ codeErrorTitle=Kód chyby\: {0}
...
+termsPlainText=Smluvní podmínky, které se mají definovat.
```
Comment (dklika) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:49]:
```suggestion
termsPlainText=Smluvní podmínky k odsouhlasení.
```

**Comment by McLaynV** [comment]
@hmlnarik you had some commits in this file (czech translation) in the past. Could you please review or merge this?

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 4]
```diff
@@ -67,11 +74,30 @@ region=Kraj
 postal_code=PSČ
 country=Stát
 emailVerified=E-mail ověřen
+website=Webová stránka
+phoneNumber=Telefoní číslo
+phoneNumberVerified=Telefoní číslo ověřeno
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:79]:
```suggestion
phoneNumber=Telefonní číslo
phoneNumberVerified=Telefonní číslo ověřeno
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 5]
```diff
@@ -67,11 +74,30 @@ region=Kraj
...
+gender=Pohlaví
+birthday=Datum narození
+zoneinfo=Časová zóna
 gssDelegationCredential=GSS Delegované Oprávnění
+logoutOtherSessions=Odhlásit se na ostatních zařízeních
 
+profileScopeConsentText=Úživatelský profil
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:86]:
```suggestion
profileScopeConsentText=Uživatelský profil
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 6]
```diff
@@ -67,11 +74,30 @@ region=Kraj
...
 country=Stát
 emailVerified=E-mail ověřen
+website=Webová stránka
+phoneNumber=Telefoní číslo
+phoneNumberVerified=Telefoní číslo ověřeno
+gender=Pohlaví
+birthday=Datum narození
+zoneinfo=Časová zóna
 gssDelegationCredential=GSS Delegované Oprávnění
+logoutOtherSessions=Odhlásit se na ostatních zařízeních
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:84]:
```suggestion
logoutOtherSessions=Odhlásit se z ostatních zařízení
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 7]
```diff
@@ -67,11 +74,30 @@ region=Kraj
...
 
+profileScopeConsentText=Úživatelský profil
+emailScopeConsentText=Emailová adresa
+addressScopeConsentText=Addresa
+phoneScopeConsentText=Telefoní číslo
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:89]:
```suggestion
phoneScopeConsentText=Telefonní číslo
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 8]
```diff
@@ -67,11 +74,30 @@ region=Kraj
...
+phoneNumberVerified=Telefoní číslo ověřeno
+gender=Pohlaví
+birthday=Datum narození
+zoneinfo=Časová zóna
 gssDelegationCredential=GSS Delegované Oprávnění
+logoutOtherSessions=Odhlásit se na ostatních zařízeních
 
+profileScopeConsentText=Úživatelský profil
+emailScopeConsentText=Emailová adresa
+addressScopeConsentText=Addresa
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:88]:
```suggestion
addressScopeConsentText=Adresa
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 9]
```diff
@@ -67,11 +74,30 @@ region=Kraj
...
+phoneScopeConsentText=Telefoní číslo
+offlineAccessScopeConsentText=Přístup offline
+samlRoleListScopeConsentText=Moje role
+rolesScopeConsentText=Uživatelské role
+
+restartLoginTooltip=Restart login
+
+loginTotpIntro=You need to set up a One Time Password generator to access this account
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:96]: Please translate

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 10]
```diff
@@ -67,11 +74,30 @@ region=Kraj
...
+phoneNumber=Telefoní číslo
+phoneNumberVerified=Telefoní číslo ověřeno
+gender=Pohlaví
+birthday=Datum narození
+zoneinfo=Časová zóna
 gssDelegationCredential=GSS Delegované Oprávnění
+logoutOtherSessions=Odhlásit se na ostatních zařízeních
 
+profileScopeConsentText=Úživatelský profil
+emailScopeConsentText=Emailová adresa
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:87]:
```suggestion
emailScopeConsentText=E-mailová adresa
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 11]
```diff
@@ -134,13 +169,19 @@ role_manage-account-links=Spravovat odkazy na účet
 role_read-token=Číst token
 role_offline-access=Přístup offline
 client_account=Účet
+client_account-console=Uživatelská Console
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:172]:
```suggestion
client_account-console=Uživatelská konzola
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 12]
```diff
@@ -185,6 +242,11 @@ emailSendErrorMessage=Nepodařilo se odeslat e-mail, zkuste to prosím později.
 accountUpdatedMessage=Váš účet byl aktualizován.
 accountPasswordUpdatedMessage=Vaše heslo bylo aktualizováno.
 
+delegationCompleteHeader=Přihlášení úspěšné
+delegationCompleteMessage=Můžete zavřít toto okno prohlížeče a vrátit se do konzolové aplikace.
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:246]:
```suggestion
delegationCompleteMessage=Můžete zavřít toto okno prohlížeče a vrátit se do aplikace.
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 13]
```diff
@@ -185,6 +242,11 @@ emailSendErrorMessage=Nepodařilo se odeslat e-mail, zkuste to prosím později.
...
+delegationFailedHeader=Přihlášení selhalo
+delegationFailedMessage=Můžete zavřít toto okno prohlížeče a vrátit se do konzolové aplikace a zkusit se znovu přihlásit.
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:248]:
```suggestion
delegationFailedMessage=Můžete zavřít toto okno prohlížeče a vrátit se do aplikace a zkusit se znovu přihlásit.
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 14]
```diff
@@ -193,6 +255,7 @@ invalidPasswordMinLowerCaseCharsMessage=Neplatné heslo: musí obsahovat minimá
 invalidPasswordMinUpperCaseCharsMessage=Neplatné heslo: musí obsahovat nejméně {0} velká písmena.
 invalidPasswordMinSpecialCharsMessage=Neplatné heslo: musí obsahovat nejméně {0} speciální znaky.
 invalidPasswordNotUsernameMessage=Neplatné heslo: nesmí být totožné s uživatelským jménem.
+invalidPasswordNotEmailMessage=Neplatné heslo: nesmí být totožné s emailovou adresou.
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:258]:
```suggestion
invalidPasswordNotEmailMessage=Neplatné heslo: nesmí být totožné s e-mailovou adresou.
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 15]
```diff
@@ -256,4 +319,76 @@ requiredAction.UPDATE_PASSWORD=Aktualizace hesla
...
+webauthn-error-different-user=První přihlášený úživatel není totožný s uživatelem přihlášeným pomocí bezpečnostního klíče.
+webauthn-error-auth-verification=Nevalidní výsledek přihlášení pomocí bezpečnostního klíče.<br/> {0}
+webauthn-error-register-verification=Nevalidní výsledek registrace bezpečnostního klíče.<br/> {0}
+webauthn-error-user-not-found=Neznámý uživatel přihlášen pomocí bezpečnostního kíče.
+
+# Identity provider
+identity-provider-redirector=Propojit s jiným poskytovatelem identit
+identity-provider-login-label=Nebo se přihlašte pomocí
+
+finalDeletionConfirmation=Pokud svůj účet odstraníte, nemůže být obnoven. Pro zachování účtu klikněte na Storno.
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:384]:
```suggestion
finalDeletionConfirmation=Pokud svůj účet odstraníte, nemůže být obnoven. Pro zachování účtu klikněte na tlačítko Zrušit.
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties [hunk 16]
```diff
@@ -256,4 +319,76 @@ requiredAction.UPDATE_PASSWORD=Aktualizace hesla
...
+webauthn-doAuthenticate=Přihlášení bezpečnostním klíčem
+
+# WebAuthn Error
+webauthn-error-title=Chyba bezpečnostního klíče
+webauthn-error-registration=Selhala registrace vašeho bezpečnostního kíče.<br/> {0}
+webauthn-error-api-get=Selhalo přihlášení pomocí bezpečnostního klíče.<br/> {0}
+webauthn-error-different-user=První přihlášený úživatel není totožný s uživatelem přihlášeným pomocí bezpečnostního klíče.
+webauthn-error-auth-verification=Nevalidní výsledek přihlášení pomocí bezpečnostního klíče.<br/> {0}
+webauthn-error-register-verification=Nevalidní výsledek registrace bezpečnostního klíče.<br/> {0}
+webauthn-error-user-not-found=Neznámý uživatel přihlášen pomocí bezpečnostního kíče.
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/login/messages/messages_cs.properties:378]:
```suggestion
webauthn-error-user-not-found=Neznámý uživatel přihlášen pomocí bezpečnostního klíče.
```

**Review by hmlnarik — CHANGES_REQUESTED** [review]
@McLaynV Thank you for the PR.
@dklika Thank you for the review.
Could you please introduce a JIRA, address the inline comments and squash the commits into a single one?

**Comment by hmlnarik** [comment]
Thank you @McLaynV
One last thing is to prefix the commit message with Keycloak JIRA, i.e. `KEYCLOAK-18913 Update messages_cs.properties`, could you please update it?

**Comment by McLaynV** [comment]
Hello @hmlnarik 
I changed the commit name as you suggested.

**Review by hmlnarik — APPROVED** [review]

**Comment by hmlnarik** [comment]
@McLaynV Thank you!

**Review by stianst — APPROVED** [review]

**Pull request merged**